src/components/App/App.js

```diff
@@ -34,34 +34,46 @@ class App extends React.Component {
      return polish.brand === newPolish.brand
    })
 
-   const index = this.state.polishes.findIndex(foundPolish => {
-     return foundPolish.brand === newPolish.brand
-   })
-   let copy = [...this.state.polishes]
-
-   if (foundPolish) {
-     const foundColor = foundPolish.colors.find(color => {
-       return color.colour_name === newPolish.colorway
-     })
-     if (!newPolish.hue && foundColor) {
-      newPolish.hue = foundColor.hex_value
+   if (!foundPolish) {
+    this.setState({collection: [...this.state.collection, newPolish]})
+    const formattedToCollection = {
+      id: newPolish.id,
+      brand: newPolish.brand,
+      image: newPolish.image,
+      colors: [
+        {
+          colorway: newPolish.colorway,
+          hue: newPolish.hue
+        }
+      ]
     }
+    return this.setState({polishes: [...this.state.polishes, formattedToCollection]})
```

Comment:
I'm not quite clear on the difference between line 50 and line 38 polishes vs collection?

Reply:
polishes is our "database" aka list of all polishes. collection is what the user owns!